setup.py

```diff
@@ -145,7 +145,7 @@
     "psutil",
     "pyyaml>=5.1",
     "pydantic",
-    "pytest>=7.2.0",
+    "pytest>=7.2.0<8.0.0",
```

Comment:
I think it requires a comma, so like

`pytest>=7.2.0,<8.0.0`

(by other lines)

Reply:
Yep, sorry, my bad. Fixed now

Reply:
`src/transformers/dependency_versions_table.py` need to be updated by running `make deps_table_update`